variables.tf

```diff
@@ -34,7 +34,17 @@ variable "sftp_users" {
       write  = optional(bool, true)
       delete = optional(bool, true)
     })
-  }))
+  }),
```

Comment:
You don't need this, just need to add an "inbound" object to the environment tfvars file you'd like the "inbound" SFTP user added in.

Reply:
done